docs/content/guide/production.ngdoc

```diff
@@ -16,10 +16,16 @@ and adds CSS classes to data-bound elements:
 - As a result of `ngBind`, `ngBindHtml` or `{{...}}` interpolations, binding data and CSS class
 `ng-binding` are attached to the corresponding element.
 
+- Whenever a value is bound to an element, that element gains a data property, `$binding`, containing an 
+array of the binding expressions
+
 - Where the compiler has created a new scope, the scope and either `ng-scope` or `ng-isolated-scope`
 CSS class are attached to the corresponding element. These scope references can then be accessed via
 `element.scope()` and `element.isolateScope()`.
 
+- Placeholder comments for directives which affect the DOM will contain information about what directive
```

Comment:
Let's change _"for directives which affect the DOM"_ to _"for structural directives"_.